qemu/tests/boot_with_different_vectors.py

```diff
@@ -78,27 +77,27 @@ def check_msi_support(session):
     def check_interrupt(session, vectors):
         error_context.context("Check the cpu interrupt of virito",
                               logging.info)
-        cmd = "cat /proc/interrupts |grep virtio"
-        output = session.cmd_output(cmd).strip()
+        ic_check_cmd = params["ic_check_cmd"]
```

Comment:
You can use "irq" or at least "int" as the abbreviate of "interrupt". "ic" is not clear enough.
Other part LGTM.

Reply:
Thanks, Xiyue. At the beginning I intended to use "ic" as the abbreviation for "interrupt controller". "int" seems to be easy to understand as an integer type, so replace "ic" with "irq".